lib/rules/web/admin_console/4.5/goto.xyaml

```diff
@@ -180,6 +180,9 @@ goto_operator_subscription_page:
   url: operatorhub/subscribe?pkg=<package_name>&catalog=<catalog_name>&catalogNamespace=openshift-marketplace&targetNamespace=<target_namespace>
 goto_one_project_subscription_page:
   url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~Subscription/<subscription_name>
+goto_csv_detail_page:
+  action: wait_box_loaded
+  url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>
```

Comment:
Should it be 
```
goto_csv_detail_page:
  url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>
  action: wait_box_loaded
```

Reply:
Also it is also added in #918 , right?

Reply:
OK, I was plan to remove the action after giving the pass log.